plugin/trino-delta-lake/src/test/java/io/trino/plugin/deltalake/TestDeltaLakeSharedGlueMetastoreWithTableRedirections.java

```diff
@@ -108,4 +122,75 @@ protected String getExpectedDeltaLakeCreateSchema(String catalogName)
                 ")";
         return format(expectedDeltaLakeCreateSchema, catalogName, schema, dataDirectory.toUri());
     }
+
+    @Test
+    public void testUnsupportedHiveTypeRedirect()
+    {
+        String tableName = "unsupported_types";
```

Comment:
The whole schema has a random name: https://github.com/trinodb/trino/blob/master/plugin/trino-delta-lake/src/test/java/io/trino/plugin/deltalake/BaseDeltaLakeSharedMetastoreWithTableRedirectionsTest.java#L28

I figured we don't need random table names, since this is not running in parallel. Also, the existing tests in this class don't use random table names.